src/apps/weblib/typeschema-api/config.yaml

```diff
@@ -76,6 +76,12 @@ definitions:
                 oneOf:
                     - type: number
                     - { type: string, enum: [auto] }
+
+    AxisChannel:
+        description: |
```

Comment:
Channel props can be inherited:
```yaml
        $extends: Channel
```

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/632/commits/bbbc5238e656e889b13c997566ff6cb833ae195d